plenum/config.py

```diff
@@ -252,6 +252,9 @@
 # Max time to wait before creating a batch for 3 phase commit
 Max3PCBatchWait = 1
 
+UPDATE_STATE_FRESHNESS = True
+STATE_FRESHNESS_WINDOW = 300  # in secs
```

Comment:
We have examples when different config entries for enable/disable and parameter are used.
I prefer more explicit approach with a separate flag for enabling/disabling, but this is very subjective.
Also if we use 0 for timeout/period/window, it may sound like this is always up-to-date, while in fact this is the opposite.

Reply:
Agree that STATE_FRESHNESS_WINDOW  may be not the best name.
I think STATE_FRESHNESS_UPDATE_INTERVAL sounds better.